modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/move_kb_object_modal.jsp

```diff
@@ -19,7 +19,7 @@ KBAdminNavigationDisplayContext kbAdminNavigationDisplayContext = new KBAdminNav
 			HashMapBuilder.<String, Object>put(
 				"items", kbAdminNavigationDisplayContext.getKBFolderDataJSONArray()
 			).put(
-				"itemToMoveParentId", kbAdminNavigationDisplayContext.getKBObjectToMoveParentId()
+				"moveParentItemId", kbAdminNavigationDisplayContext.getMoveParentKBObjectId()
```

Comment:
There is a reason to not call `moveParentKBObjectId` instead `moveParentItemId`
Usually is :
**name** = get**Name**

Reply:
No, I'll fix it! :)